### PR TITLE
#165485325 Implements user dashboard with article statistics

### PR DESCRIPTION
**Pull request by valenstical**
#### What does this PR do?
This PR implements a feature that adds a user dashboard page to the application with statistics about the author's articles.

#### Description of Task to be completed

- [x] Write unit test
- [x] Create presentational and container Dashboard components
- [x] Create a Summary component to display stats about the articles
- [x] Add dashboard container component to the system route
- [x] Create relevant selectors, actions, and reducers for the dashboard component
- [x] Consume API endpoint to fetch all articles by the current authenticated user
- [x] Write integration tests

#### How should this be manually tested

1. Clone repo by running the following command on your terminal:

```bash
git clone -b feature/165485325/view-article-stats--single-branch \
https://github.com/andela/apollo-ah-frontend.git
```

2. Navigate into the repo directory: `cd apollo-ah-frontend`

2. Install dependency packages with: `npm install`

3. On the root directory, create a `.env` file from `.env.example` and provide required credentials

4. Run `npm test`

#### Any background context you want to provide
N/A

#### What are the relevant pivotal tracker stories

[#165485325](https://www.pivotaltracker.com/story/show/165485325)

#### Screenshots
ezgif-2-bfdc4cdc39e7

ezgif-4-7ab3f66ad890

### CODE REVIEW

File: src/test/actions/dashboardAction.test.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import '@babel/polyfill';
```
Comment (devdbrandy) [inline src/test/actions/dashboardAction.test.js:1]: We don't have to import `@babel/polyfill` in all our test files; app is configured to use `transform-runtime`, which provides same features as polyfill.

**Review by devdbrandy — CHANGES_REQUESTED** [review]
Good job @valenstical, we now have user dashboard with article statistics ✨. Kindly attend to my requested changes.

File: src/test/actions/dashboardAction.test.js (hunk 1 above)
Comment (valenstical) [reply src/test/actions/dashboardAction.test.js:1]: Great. Thanks for pointing that out. I have removed all occurrences of that import.

**Review by devdbrandy — APPROVED** [review]

**Comment by aanchirinah** [comment]
@valenstical , Build failing

**Comment by devdbrandy** [comment]
Hi @aanchirinah, issue concerning the failing build has been resolved.

**Review by aanchirinah — APPROVED** [review]
LGTM

**Pull request merged**